doc/nrf/config_and_build/bootloaders/bootloader_external_flash.rst

```diff
@@ -13,14 +13,24 @@ When using MCUboot, you can store the storage partition for the secondary slot i
 * Single-byte read and write.
 * Writing data from the internal flash memory to the external flash memory.
 
+To enable external flash with MCUboot the following must take place:
+
+1. Follow the instructions in :ref:`pm_external_flash`, which enables external flash use in the nRF5340 DK's DTS file.
+
+#. Enable the :kconfig:option:`CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY` Kconfig option.
+   Depending on the build, this option is set to ``y`` by default.
+
+#. Adjust the ``CONFIG_BOOT_MAX_IMG_SECTORS`` `MCUboot Kconfig option <https://github.com/nrfconnect/sdk-mcuboot/blob/main/boot/zephyr/Kconfig#L370>`_ accordingly for child images, which defines the maximum number of image sectors MCUboot can handle, as MCUboot typically increases slot sizes when external flash is enabled.
```

Comment:
`image sectors MCUboot can handle`
this is for any image-slot. mayby worth to describe that `image sectors` size are the same as flash erase-block-size for any memory integrated in NCS.
(MCUboot uses different terminology)